.prettierrc.json

```diff
@@ -8,3 +8,5 @@
   "arrowParens": "avoid",
   "proseWrap": "always"
 }
+
```

Comment:
a co to? :)

Reply:
Nie mam pojęcia. Nie  dodawałam nic takiego, chyba, że nieświadomie ;)

Reply:
dodałaś po prostu następny wiersz tylko